<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>200</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string extracomment="Mam WatchDog Program">MAM WatchDog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {background:#64BCC4;border-style:solid;border-width:1px;border-color:rgb(180,180,180);} </string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>157</width>
          <height>69</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>157</width>
          <height>69</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{background:url(&quot;./resources/mam.png&quot;);border:none;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{background:url(&quot;./resources/closeHover.png&quot;);border:none;}
QPushButton:pressed{background:url(&quot;./resources/closeHover.png&quot;);border:none;}
QPushButton{background:url(&quot;./resources/close.png&quot;);border:none;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MAM</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(242,242,242),stop:1 rgb(225,225,225));border-style:solid;border-color:gray;border-width:1px;border-radius:4px}
QPushButton:hover {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(242,242,242),stop:1 rgb(237,237,237));border-style:solid;border-color:gray;border-width:1px;border-radius:4px}
QPushButton:pressed {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(200,200,200),stop:1 #EEEEEE);border-style:solid;border-color:gray;border-width:1px;border-radius:4px}</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(242,242,242),stop:1 rgb(225,225,225));border-style:solid;border-color:gray;border-width:1px;border-radius:4px}
QPushButton:hover {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(242,242,242),stop:1 rgb(237,237,237));border-style:solid;border-color:gray;border-width:1px;border-radius:4px}
QPushButton:pressed {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(200,200,200),stop:1 #EEEEEE);border-style:solid;border-color:gray;border-width:1px;border-radius:4px}</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>MAS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(242,242,242),stop:1 rgb(225,225,225));border-style:solid;border-color:gray;border-width:1px;border-radius:4px}
QPushButton:hover {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(242,242,242),stop:1 rgb(237,237,237));border-style:solid;border-color:gray;border-width:1px;border-radius:4px}
QPushButton:pressed {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(200,200,200),stop:1 #EEEEEE);border-style:solid;border-color:gray;border-width:1px;border-radius:4px}</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(242,242,242),stop:1 rgb(225,225,225));border-style:solid;border-color:gray;border-width:1px;border-radius:4px}
QPushButton:hover {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(242,242,242),stop:1 rgb(237,237,237));border-style:solid;border-color:gray;border-width:1px;border-radius:4px}
QPushButton:pressed {background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FFFFFF,stop:0.5 rgb(200,200,200),stop:1 #EEEEEE);border-style:solid;border-color:gray;border-width:1px;border-radius:4px}</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
